concourse/tasks/delete_admin.yml

```diff
@@ -3,32 +3,31 @@ platform: linux
 image_resource:
   type: docker-image
   source:
-    repository: governmentpaas/cf-uaac
+    repository: governmentpaas/cf-cli
 inputs:
   - name: paas-cf
   - name: cf-secrets
   - name: cf-manifest
   - name: admin-creds
   - name: bosh-CA
+  - name: config
 run:
   path: sh
   args:
     - -e
     - -c
     - |
-      NAME=$(cat admin-creds/username)
       if [ "${ENABLE_ADMIN_USER_CREATION}" == "false" ]; then
         echo "Temporary user creation is disabled (ENABLE_ADMIN_USER_CREATION=${ENABLE_ADMIN_USER_CREATION}). Skipping."
         exit 0
       fi
       ./paas-cf/concourse/scripts/import_bosh_ca.sh
+      . ./config/config.sh
```

Comment:
That was a suggestion that we don't need to run a separate "get credentials" container/task every time we need to run this, but instead of running config.sh (produced by that container) that exports the variables above, we can export them directly here...